dist/index.html

```diff
@@ -57,6 +57,222 @@ <h2 href="#" class="connect">LET'S CONNECT</h2>
+                                <p class="role">Back End Dev</p>
+                                <img src="./img/Counter.png" alt="Counter">
+                                <p class="role">2023</p>
+                              </div>
+                           <p class="card-description">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                           <ul class="small-btn">
+                             <li class="tag-description">HTML</li>
+                             <li class="tag-description">CSS</li>
+                             <li class="tag-description">JAVASCRIPT</li>
+                           </ul>
+                           <button type="button" class="project-btn">See project</button>
+                         </div>
+                    </div>
+                  </div> 
+          
+                  <div class="grid-container card-4">
+                    <div class="card-container">
+                      <div>
+                        <img class="card-img" src="./img/projects/Snapshoot Portfolio_4.png" alt="photo of my 1st project" />
+                      </div>
+                         <div class="text-container">
+                           <h2 class="card-title">Multi-Post <br> Stories</h3>
+                              <div class="frame-two">
+                                <p class="client">CANOPY</p>
+                                <img src="./img/Counter.png" alt="Counter">
+                                <p class="role">Back End Dev</p>
+                                <img src="./img/Counter.png" alt="Counter">
+                                <p class="role">2023</p>
+                              </div>
+                           <p class="card-description">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                           <ul class="small-btn">
+                             <li class="tag-description">HTML</li>
+                             <li class="tag-description">CSS</li>
+                             <li class="tag-description">JAVASCRIPT</li>
+                           </ul>
+                           <button type="button" class="project-btn">See project</button>
+                         </div>
+                    </div>
+                  </div> 
+            </section>
+
+            <section  class="my-skills">
+                <div class="about "> 
+                    <div class="about-self">
+                        <div><h2 class="heading">About  Myself</h2></div>
+                        <div><p class="info">Hello I'm a software developer! I can help you build a product, feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don't hesitate to contact me.</p></div>
+                        <h3 href="#" class="connect">LET'S CONNECT</h3>
+                   <ul class="social-media">
```

Comment:
- [ ] Kindly use the font-family used in the design for every text on this section as I've mentioned above about the `Poppins` font that you need to import from google fonts.

### Your Design
![your-let-connect](https://user-images.githubusercontent.com/26597432/234845208-f2db61e6-6d69-4116-8361-d3c41c549bd2.png)


### Figma Design
![temp-let-connect](https://user-images.githubusercontent.com/26597432/234844885-4e65fe63-67bc-4acb-8ee7-6b57de716721.png)